### ISSUE TITLE
North Macedonia is incorrectly listed as its old name Macedonia

### ISSUE THREAD

**Issue by northmacedonianfan38572** [teuteuf/worldle#84]
North Macedonia is incorrectly listed as Macedonia. The name changed to North Macedonia in 2019!

https://www.theguardian.com/world/2019/feb/12/nato-flag-raised-ahead-of-north-macedonias-prospective-accession

https://en.wikipedia.org/wiki/North_Macedonia

**Comment by Sun8908** [comment]
Yes! The ISO 3166-1 list, according to [Wikipedia](https://en.wikipedia.org/wiki/ISO_3166-1), has already changed a lot.

The current source from [Google](https://developers.google.com/public-data/docs/canonical/countries_csv) hasn't been updated for a long time.

Netherlands Antilles which was disestablished more than 10 years ago is still in the game. And many Caribbean territories are not listed, including Dutch territories: Bonaire, Sint Eustatius and Saba (Caribbean Netherlands), and Sint Maarten; and French territories: Saint Martin, and Saint Barthélemy.

Meanwhile, Gaza Strip, which is not on the current ISO 3166-1 list according to Wikipedia, is also still in the game.

**Comment by teuteuf** [comment]
Thanks for reporting it!
I fixed North Macedonia but keep this issue open to keep track of @Sun8908 comment and fix it later.
Indeed, this google csv is a bit too old...
